spyder/plugins/editor/plugin.py

```diff
@@ -193,6 +193,8 @@ def setup_page(self):
         run_selection_group = QGroupBox(_("Run selection"))
         focus_box = newcb(_("Maintain focus in the Editor after running cells "
                             "or selections"), 'focus_to_editor')
+        run_cell_box = newcb(_("Use the run cell function to execute code "
+                               "cells"), 'run_cell_func')

```

Comment:
The `runcell()` function is just the implementation of not having the full cell contents in the history and the console, so it likely doesn't need to be mentioned

For more clarity, I can flip the setting in the code and go for:

`Copy full contents to the console when running code cells`

With the setting unchecked by default.

The tool tip could read:

`When running a code cell the full contents of the cell will be copied to the console and the history`

Reply:
> The runcell() function is just the implementation of not having the full cell contents in the history and the console, so it likely doesn't need to be mentioned

Agreed, that's my opinion as well.

> Copy full contents to the console when running code cells

I guess that could be a little clearer, although I don't feel that strongly a about it. As a minor refinement, I'd suggest ``Show full cell contents in the console when running code cells``. While I'm normally very opposed to repetition, I think its necessary here for clarity. Regarding the tooltip, a suggested revision would be ``Print code cell contents to the Console and include them in the History Log when running cells``.

Reply:
I also agree with

> Copy full cell contents to the console when running code cells

and making the option `False`. Thanks @bcolsen!

Reply:
I think I'll go for:

`Copy full cell contents to the console when running code cells`

I like 'copy' in there because it's more than just shown or printed into the console, it's more like what would happen if you copied and pasted the cell into the console manually.

Reply:
+1 from me.